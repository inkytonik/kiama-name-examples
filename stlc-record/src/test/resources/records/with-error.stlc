with 1 do 2
