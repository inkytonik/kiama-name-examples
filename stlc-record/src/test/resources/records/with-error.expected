src/test/resources/records/with-error.stlc:1:1:info: with 1 do  ...: unknown
with 1 do 2
^
src/test/resources/records/with-error.stlc:1:6:info: 1: num
with 1 do 2
     ^
src/test/resources/records/with-error.stlc:1:11:info: 2: num
with 1 do 2
          ^
unknown
